mail/mail.services.ts

```diff
@@ -0,0 +1,16 @@
+import { instance } from "../common/axios.client";
+
+class MailService {
+  public static async sendMail(
+    listId: string,
+    title: string,
+    content: string
+  ): Promise<void> {
+    const body: URLSearchParams = new URLSearchParams();
+    body.append('uuid', listId || '');
+
+    console.log((await instance.post('/lists', body)).data);
```

Comment:
Move call to the list microservice to the list module (list service)

Reply:
Moved